src/components/PlantingSites/PlantingSiteWithMapHelpModal.tsx

```diff
@@ -0,0 +1,36 @@
+import strings from 'src/strings';
+import Button from 'src/components/common/button/Button';
+import React from 'react';
+import DialogBox from 'src/components/common/DialogBox/DialogBox';
+import { Typography } from '@mui/material';
+import { TERRAWARE_SUPPORT_LINK } from 'src/constants';
+import parseHtml from 'html-react-parser';
+
+export type PlantingSiteWithMapHelpModalProps = {
+  open: boolean;
+  onClose: () => void;
+};
+
+export default function PlantingSiteWithMapHelpModal(props: PlantingSiteWithMapHelpModalProps): JSX.Element {
+  const { open, onClose } = props;
+
+  return (
+    <DialogBox
+      scrolled
+      onClose={onClose}
+      open={open}
+      title={strings.SELECT_PLANTING_SITE_TYPE}
+      size={'medium'}
+      middleButtons={[<Button onClick={onClose} id='done' label={strings.DONE} key='button-1' />]}
+    >
+      <Typography>
+        {parseHtml(
```

Comment:
this is cool!
fwiw, see https://terraformation.atlassian.net/wiki/spaces/PM/pages/708280327/PRD+Support+Spanish+localization#Stop-mixing-link-and-non-link-words-in-phrases-(preferred-approach) for preferred approach.. not sure if it's less of an issue here but just to check

Reply:
This was the cleanest way to add the link inline. The preferred approach would involve adding a button or something

Reply:
from a translation standpoint we are looking at, "Please _{blank}_ to create a site..." where the {blank} is some call-to-action ("contact us" in this case). Maybe this doesn't translate well, or lend itself to other grammars, I'm not sure.

Reply:
Got it.. the preferred approach is to link the entire sentence. We can revisit this when we get to i18n for sure.
Also would be good to have a 'contact us' string that is already lower cased.. ideally we use translated strings as-is.

Reply:
Oh gotcha. thanks

Reply:
My bad.. ignore my previous comment. 
The string replace approach may break in general - depending on the language. As an example - “contact us” can theoretically be contracted when used in a full sentence.. in some language. 
Let’s defer all this for i18n discussions.
Code looks great as-is for now.